src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -131,13 +141,7 @@ class TransportManagementService implements Serializable {
             validResponseCodes    : "100:399"
         ]
 
-        def response = script.httpRequest(defaultParameters + parameters)
-
-        if (config.verbose) {
-            echo("Received response '${response.content}' with status ${response.status}.")
-        }
-
-        return response.content
+        script.httpRequest(defaultParameters + parameters)
```

Comment:
Does the method return an http response without explicitly specifying a return statement?

Reply:
yes, it's groovy behaviour. The return value of the latest executed statement is returned also without having the `return´ keyword.